### ISSUE TITLE
Getting battle.net accounts that don't own the game into a clan (for clan expansion)

### ISSUE THREAD

**Issue by ghost** [Bungie-net/api#769]
I used to be able to make new battle.net accounts and make them clan leaders without needing the game, but several months back that changed.  I was hoping to take advantage of the free game deal right now, but it requires SMS protection to accept the gift for accounts created after the announcement.  I asked Blizzard if there was a way to waive that requirement on their end and they said they couldn't, so now my hope is that there is something Bungie can do here.

I know this isn't the place to ask, but I don't have a contact at Bungie for general questions.  The forums seem pointless (I've had bad experiences).  Is it possible to get a contact at Bungie?  I've got nearly 2300 destiny players in my discord partnered server and 7 clans full of very active players, so I'm hoping that is enough to get someones email address for this sort of question!

(also sorry for the spam if questions... Figured these things should have separate tickets so they are more useful to others)

**Comment by justrealmilk** [comment]
I suspect the 2FA requirement is to prevent humans from claiming the gift several times over and later selling the accounts

**Comment by Akumati1435** [comment]
Yep, it makes perfect sense to do that.  I'm one of maybe 10 people that would have an legitimate request to ask for an exemption though.

**Comment by soren42** [comment]
Akumati1435, I know it’s a little ornerous, but have you considered just creating a Google account for each?  That would give you a unique email address and, through Google Voice, a phone number that can receive texts.

**Comment by vthornheart-bng** [comment]
Hang tight before you do something like what @soren42 suggested - I still need to get more information before I can give you a good response.  But in the interim I don't want you doing something that'll get you in trouble with Blizzard!

**Comment by vthornheart-bng** [comment]
Okay, I talked to some folks - unfortunately we can’t help you with this: doing so would circumvent Blizzard account creation policy.

What's the use case?  Maybe there's some other solution?  (for instance, taking someone who you know wants to be in the clan and using their account?)

**Comment by Akumati1435** [comment]
Do you have a link to their account creation policy?  I searched their [EULA](https://www.blizzard.com/en-us/legal/fba4d00f-c7e4-4883-b8b9-1b4500a402ea/blizzard-end-user-license-agreement) for the word "account" before calling blizzard to make sure they couldn't fire back on me and kill my existing accounts when I tried getting an exemption to the 2FA requirement for the free game. The only account quantity limitation I could find was just for accounts with blizzard balances.  **It actually specifically says you can play on multiple accounts.**

"Blizzard Balance is not a bank account. And, **while you can register and play on multiple Accounts**, you are not allowed to have more than three (3) Accounts with Blizzard Balance."

A reddit post had a quote from their docs about having a 3 account limit per person, but I don't see that anywhere in the EULA.

The use case is that I'm cautious with giving people control of my clans. People who I make leaders and give an entire clan to need to prove they have the community's interests in mind because they could disappear or try to ninja a clan if they choose to.   Officers can be kicked out, but leaders can kick YOU out.  The other thing is that I have accounts in each clan for my clan's admin tools.  I could have one of my officers/leaders log in so I can store a token with clan admin privs, but if anything goes wrong, it's better if I have an account in the clan so I can fix things myself.

Another factor is the high turnover rate for Destiny players.  People like the game one day, then the next they are tired of the game and quit.  In the past month I was made leader of 3 more of my clans because the leaders quit the game.  It's also helpful to be in the clans because I don't have everything automated yet.

**Comment by vthornheart-bng** [comment]
The use case is interesting, and I can definitely see where you'd want to have more control if you've got those concerns.  But unfortunately I don't think we're in a position where we can help. :(  The folks I hit up over here said as much to me as well, and asked me to relay it on.

Sorry I can't help in this situation, but my hands are tied on this.  The best people - really, the only people - who can actually help you with this in a fully legitimate way is Blizzard themselves.  I can't promise that they *will*, but they're the only game in town for resolving this particular situation.  You'll have to hit up their support and make the request: perhaps if you explain the specifics of your situation (the concerns you mentioned above about control of the Clans and management of inactive users) they'll give you an exemption from this SMS verification policy?

**Comment by Akumati1435** [comment]
That's exactly what I asked and they said they cannot help.
